docs/README.md

```diff
@@ -1,8 +1,8 @@
-# Mend Renovate Community Edition and Enterprise Edition Documentation
+# Mend Renovate Self-hosted Documentation
```

Comment:
Mend Renovate CLI is also self-hosted, so this title is too wide

Reply:
Updated all references to "Mend Renovate Self-hosted App(s)"